Add include_origin option for PeriodicBuffer::compute (#942)
glotzerlab/freud: ChangeLog.md

### Added
* (breaking) Some `freud.diffraction.StaticStructureFactorDebye` property names changed to be more descriptive.

> * `freud.locality.PeriodicBuffer` now has an optional `include_input_points` to include the provided system points in the buffer. This defaults to false as to not break behavior.

Reviewer: ```suggestion
* `include_input_points` argument to `freud.locality.PeriodicBuffer`. 
```
If users want all the details on this they can just look it up in the documentation.

### Changed

> * `freud.data.Unitcell.generate_system` now generates positions so that the same lattice site's project positions are contiguous.

Reviewer: ```suggestion
* `freud.data.Unitcell.generate_system` now generates positions in the same order as the basis positions.
```
I think this is a clearer, more concise way to word this. Feel free to suggest other options.

### Added
* (breaking) Some `freud.diffraction.StaticStructureFactorDebye` property names changed to be more descriptive.

### Changed

> * `freud.data.Unitcell.generate_system` now generates lattice site positions in a contiguously.

Reviewer 2: Not sure what 'in a contiguously' means.

> 

Reviewer: This doesn't quite make sense.

Author: correct it doesn't :sweat_smile:
